std/ascii.d

```diff
@@ -136,13 +186,23 @@ unittest
 
 
 /++
-    Returns whether $(D c) is a digit (0..9).
+    Params: c = The character to test.
+    Returns: Whether $(D c) is a digit (0..9).
   +/
 bool isDigit(dchar c) @safe pure nothrow @nogc
 {
     return '0' <= c && c <= '9';
 }
 
+///
+unittest
+{
+    assert( isDigit('3'));
+    assert( isDigit('8'));
+    assert(!isDigit('B'));
+    assert(!isDigit('#'));
+}
```

Comment:
Considering the rigorous mentioning of behaviour on Unicode characters: Unicode digits like full-width digits (０１２３４５６７８９) do exist!